react/Layout/Layout.md

```diff
@@ -2,51 +2,90 @@ The Layout component brings a strong context for apps with any screen resolution
 
 * `<Main />` is considered the wrapper of your app content. `<Content />` should be a direct children of it but you can add other components as well, like a toolbar and such.
 * `<Content />` is the main content or your app.
+* Secondary `<NavItem />` are not visible on mobile
```

Comment:
```suggestion
⚠️  Secondary `<NavItem />` are not visible on mobile
```
 ?

Reply:
Also, we display none on +medium-screen. We should tell so, this is not just on mobile right? Tablets are also concerned?

Reply:
Yes tablets are also concerned, I updated the doc to make it clear and added the warning icon.